src/rider/main/kotlin/me/seclerp/rider/plugins/efcore/features/database/update/UpdateDatabaseDialogWrapper.kt

```diff
@@ -68,13 +82,11 @@ class UpdateDatabaseDialogWrapper(
     // UI
     override fun Panel.createPrimaryOptions() {
         row("Target migration:") {
-            textFieldWithCompletion(dataCtx.targetMigration, currentDbContextMigrationsList, intellijProject, completionItemsIcon)
-                .horizontalAlign(HorizontalAlign.FILL)
-                .comment("Use <code>0</code> as a target migration to undo all applied migrations")
-                .focused()
-                .validationOnInput(validator.targetMigrationValidation())
-                .validationOnApply(validator.targetMigrationValidation())
-                .monospaced()
+            iconComboBox(migrationView, currentDbContextMigrationsList)
+                    .validationOnInput(validator.targetMigrationValidation())
+                    .validationOnApply(validator.targetMigrationValidation())
+                    .comment("Use <code>null</code> as a target migration to undo all applied migrations")
```

Comment:
Let's leave the `0` case here. It's more convenient because it is also used in CLI to represent "before first" migration. Please take a look into "Generate SQL Script" dialog for example how to accomplish that